packages/vapor/scss/components/lightbox-search.scss

```diff
@@ -1,9 +1,11 @@
 .lightbox-search-input {
     width: 100%;
     padding: 10px 20px;
-    color: var(--deprecated-dark-grey);
+    color: var(--grey-60);
+    font-weight: var(--font-weight-book);
     font-size: 2rem;
     border: none;
+    border-radius: 8px;
```

Comment:
it would be good to have variables for the border radius in vapor for homogeneity ;)